src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Get-TaskResult.ps1

```diff
@@ -28,43 +28,29 @@ Microsoft.PowerShell.Core\Set-StrictMode -Version Latest
 function Get-TaskResult {
```

Comment:
Do not touch this file.  Please revert any changes here to whatever is already in Preview.

Reply:
Sure